src/files/compose/readCompose.ts

```diff
@@ -2,23 +2,39 @@ import yaml from "js-yaml";
 import { readFile } from "../../utils/file.js";
 import { getComposePath } from "./getComposePath.js";
 import { ComposePaths, Compose } from "@dappnode/types";
+import { merge } from "lodash-es";
 
 /**
- * Read a compose parsed data
- * Without arguments defaults to write the manifest at './docker-compose.yml'
- * @return compose object
+ * Read a compose file and optionally merge it with a variant compose file.
+ * @param paths The primary compose file paths.
+ * @param variantPaths Optional variant compose file paths to merge with the primary compose.
```

Comment:
readCompose only has `paths` as parameter, it doesnt have `variantPaths`. Maybe desc is outdated?